app/main/server/app/ui/main-window.js

```diff
@@ -81,6 +83,48 @@ module.exports = class MainWindow {
       this.hide(true);
     });
 
+    // process list of available displays into user-friendly list
+    const displays = electron.screen.getAllDisplays();
```

Comment:
This should be asynchronously handled so that it doesn't slow the load time.  new Promise()...

Reply:
I can add this, but I just timed it and it takes 1.1ms

Reply:
1.1ms isn't a big deal, but this plus other startup items can add up over time, no worries though.